docs/api_docs/getting_started_with_apis.rst

```diff
@@ -0,0 +1,39 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _getting_started_with_apis:
+
+***************************************
+Getting Started with Vertiq's APIs
+***************************************
+
+What is the Purpose of Vertiq's APIs
+===========================================
+Vertiq's APIs provide a simple mechanism for communicating with your modules via the IQUART protocol. IQUART is Vertiq's proprietary serial 
+protocol meant to allow configuration and control of all Vertiq modules. To learn more about IQUART, please see :ref:`the protocol's documentation <uart_messaging>`. 
+Our APIs provide support for Python, Matlab, C++, and Arduino. These options provide flexibility to work easily on both PC-based and embedded solutions.
+
+If you intend to use the API specifically in a PC based environment, we recommend using either the Python or Matlab APIs.
```

Comment:
This may be a bit limiting. You can use the C++ one on the PC, and it can actually be quite a bit faster than the Python one in my experience, so there are cases where people may want to do that. I'd say either ditch this altogether, or say something closer to "Python or Matlab are generally easier to get started with, though it is also possible to use C++ on the PC"